installer/kvm-qemu.sh

```diff
@@ -738,6 +738,7 @@ function install_virt_manager() {
         echo "[+] Cloned Virt Manager repo"
     fi
     cd "virt-manager" || return
+    git checkout v4.1.0
```

Comment:
We should probably checkout a sha and have an env var to set that sha as we do with other projects here.